Dockerfiles/gadget-default.Dockerfile

```diff
@@ -6,7 +6,7 @@ ARG BUILDER_IMAGE=debian:bullseye
 
 # BCC built from the gadget branch in the kinvolk/bcc fork.
 # See BCC section in docs/CONTRIBUTING.md for further details.
-ARG BCC="quay.io/kinvolk/bcc:a432665d57f5fe10e30e6420208462711f865d5f-focal-release"
+ARG BCC="quay.io/kinvolk/bcc:401c6c81f131d539e49646ab337d3a738c7be47d-focal-release"
```

Comment:
I forced the gadget-collector to call the standard version instead of the CO-RE:
```golang
	// t.tracer, err = tracer.NewTracer(config, t.helpers, eventCallback)
	// if err != nil {
	// 	trace.Status.OperationWarning = fmt.Sprint("failed to create core tracer. Falling back to standard one")

	// fallback to standard tracer
	log.Infof("Gadget %s: falling back to standard tracer. CO-RE tracer failed: %s",
		trace.Spec.Gadget, err)

	t.tracer, err = standardtracer.NewTracer(config, eventCallback)
	if err != nil {
		trace.Status.OperationError = fmt.Sprintf("failed to create tracer: %s", err)
		return
	}
	// }
```

And I am getting this error:


```bash
$ ./kubectl-gadget trace bind -A
NODE             NAMESPACE        POD                            CONTAINER        PID     COMM             PROTO  ADDR             PORT    OPTS    IF
warn: node worker: failed to unmarshal event '{"node": "worker", "uid": 0, "pid": 663508, "comm": "nc", "pod": "test-pod", "port": 4242, "if": 0, "container": "test-pod", "addr": "::", "proto": "UNKNOWN", "namespace": "default", "time": 0.0, "opts": {"R": true, "N": false, "r": false, "T": false, "F": false}}': json: cannot unmarshal number into Go struct field Event.if of type string
```

I generated the event following the gadget's guide:

```bash
$ kubectl exec -ti test-pod -- nc -l -p 4242 -w 1
```

Is it working for you?

Reply:
The BCC tool is working correctly as we tested in the BCC PR, but the "if" field is indeed not a string:

```bash
$ /usr/share/bcc/tools/bindsnoop --json
{"comm": "nc", "uid": 0, "if": 0, "time": 0.0, "proto": "UNKNOWN", "pid": 679066, "port": 4242, "opts": {"R": true, "N": false, "r": false, "T": false, "F": false}, "addr": "::"}
```

Reply:
Ah, it's broken from the beginning I think -> https://github.com/kinvolk/inspektor-gadget/pull/523.

Reply:
I opened https://github.com/kinvolk/bcc/pull/24 and updated this PR to fix that issue as well.

Reply:
It is working now. Thanks!